src/platform/Linux/CHIPBluezHelper.cpp

```diff
@@ -48,9 +48,11 @@
  *          Provides Bluez dbus implementatioon for BLE
  */
 
+#include <AdditionalDataPayload.h>
```

Comment:
I'd think this would be `#include <setup_payload/AdditionalDataPayload.h>`.

Reply:
i thought the same at first, but it turns out the sources in build.gn is including all the files under setup_payload
i compiled it on linux for further verification

Reply:
Yes, but we shouldn't be doing that, even if we are right now.  Should be including things rooted at `src`

Reply:
I agree, i see this problem in alot of build files, i added an issue to track this problem, #4257